### PR TITLE
Show author list for events in recent discussion

### PR DESCRIPTION
**Pull request by Will-Howard**
Before:
Screenshot 2023-10-17 at 13 25 29

After:
Screenshot 2023-10-17 at 13 24 04

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1205738981569676) by [Unito](https://www.unito.io)

### CODE REVIEW

**Review by oetherington — APPROVED** [review]
LGTM - FYI @agnesstenlund

**Comment by agnesstenlund** [comment]
Great! 🙌

File: packages/lesswrong/components/ea-forum/EAPostMeta.tsx [hunk 1]
```diff
@@ -34,7 +34,13 @@ const styles = (theme: ThemeType) => ({
     width: 16,
   },
   eventOrganizer: {
-    marginLeft: 4,
+    display: "flex",
+    // Unfortunately this !important is necessary because the root class in TruncatedAuthorsList overrides
+    // with a 13px font size even if you pass in a className directly
```
[the remark this answers is not in the document]
Comment (jpaddison3) [reply packages/lesswrong/components/ea-forum/EAPostMeta.tsx:39]: The idiomatic way in our casebase is to change style priorities. Agree debatable if its better.

[the remark this answers is not in the document]
Comment (Will-Howard) [reply packages/lesswrong/components/ea-forum/EAPostMeta.tsx]: I've changed it 👍

**Pull request merged**